config/initializers/feature_flags.rb

```diff
@@ -0,0 +1,5 @@
+# Every time a content item is published, content-store updates Router's
+# backend address for that content item's rendering app.
+# If the environment variable NO_SET_ROUTER_BACKEND_ADDRESS_ON_PUBLISH is set,
+# content-store will not update backend addresses on publish.
+SET_ROUTER_BACKEND_ADDRESS_ON_PUBLISH = !ENV["NO_SET_ROUTER_BACKEND_ADDRESS_ON_PUBLISH"]
```

Comment:
Since we don't have a common concept of `config/initializers/feature_flags` I'd imagine it's better to have this just directly in app/models/route_set.rb and avoid the need to search the codebase for where the constant is set and to learn more.

I'd also include in the comment that this is added temporarily for the 2020 replatforming project and is expected to not be needed once that is complete.

Reply:
Ah ok, fair enough. I'll do that.